runtime/js/99_main.js

```diff
@@ -404,13 +404,12 @@ function promiseRejectMacrotaskCallback() {
 
 let hasBootstrapped = false;
 // Set up global properties shared by main and worker runtime.
+delete globalThis.console;
```

Comment:
Why is console getting deleted here?

Reply:
This is the `console` object that V8 adds onto the global wrapper object on context creation. We don't want this console object to shadow the console object exposed by the globalThis proxy.

Reply:
Maybe we should add that information to a comment?